src/common/offline_transformations/include/mask_attribute.hpp

```diff
@@ -203,4 +210,10 @@ Mask::Ptr getMask(const Output<Node> & output);
 
 void setMask(Output<Node> output, const Mask::Ptr & mask);
 
+void setResultMask(Output<Node> output, const Mask::Ptr & mask);
+
+Mask::Ptr getInitMask(const Output<Node> & output);
```

Comment:
It looks like the initialization mask is only used for debugging. If so, create an initialization mask only in debug mode.

Reply:
@GlebKazantaev, could you please guide me which key I should use to check if ov compiled in debug mode?

Reply:
Usually it is `ENABLE_OPENVINO_DEBUG`

Reply:
Also please add description to this functoins.

Reply:
Thanks! Please check